requirements.txt

```diff
@@ -13,3 +13,5 @@ sentry-sdk==0.14.4
 django-contrib-comments==1.9.2
 django-s3-storage==0.13.2
 watchtower==0.7.3
+arrow==0.14.7
```

Comment:
make sure arrow is really need

Reply:
lets discuss @ https://github.com/asaf-kali/coursist/pull/113#discussion_r452329604